.github/workflows/newline.yml

```diff
@@ -0,0 +1,13 @@
+name: Final new line
+on:
+  pull_request:
+    types: [opened, synchronize]
+jobs:
+  build:
+    name: New line
+    runs-on: ubuntu-16.04
+    steps:
+      - uses: actions/checkout@master
+      - uses: Logerfo/newline-action@0.0.2
```

Comment:
the version of this external, unvetted, action will have to be pinned to a sha

Reply:
```suggestion
      - uses: Logerfo/newline-action@d5cb021020d383c7f38802dce86ce4bf6f14a74f
```